packages/world/src/modules/init/constants.sol

```diff
@@ -37,3 +37,11 @@ ResourceId constant BATCH_CALL_SYSTEM_ID = ResourceId.wrap(
 ResourceId constant REGISTRATION_SYSTEM_ID = ResourceId.wrap(
   bytes32(abi.encodePacked(RESOURCE_SYSTEM, ROOT_NAMESPACE, bytes16("Registration")))
 );
+
+/**
+ * @dev Resource ID for the call with signature system.
+ * @dev This ID is derived from the RESOURCE_SYSTEM type, the ROOT_NAMESPACE, and the system name.
+ */
+ResourceId constant CALL_WITH_SIGNATURE_SYSTEM_ID = ResourceId.wrap(
+  (bytes32(abi.encodePacked(RESOURCE_SYSTEM, ROOT_NAMESPACE, "CallWithSignature")))
```

Comment:
```suggestion
  (bytes32(abi.encodePacked(RESOURCE_SYSTEM, ROOT_NAMESPACE, bytes16("CallWithSignatur"))))
```
`bytes16` to keep us honest

and snipped off the last char because the full name didn't fit into bytes16